app/controlplane/pkg/authz/authz.go

```diff
@@ -255,6 +265,11 @@ var rolesMap = map[Role][]*Policy{
 		// integrations
 		PolicyAttachedIntegrationAttach,
 		PolicyAttachedIntegrationDetach,
+
+		// Project API Token
+		PolicyProjectAPITokenList,
+		PolicyProjectAPITokenCreate,
+		PolicyProjectAPITokenRevoke,
```

Comment:
Should these two be enabled for Project Viewers? They could escalate privileges, right?

Reply:
they are not, I think Github is messing with the visualization

Reply:
right.